tests/codemods/test_fix_task_instantiation.py

```diff
@@ -0,0 +1,195 @@
+                    print(args)
+
+                my_coro = coro(1, 2)
+                asyncio.create_task(my_coro)
+                """,
+            ),
+            (
+                """
+                import asyncio
+                my_loop = asyncio.get_event_loop()
+                asyncio.Task(coro(1, 2), loop=my_loop)
+                """,
+                """
+                import asyncio
+                my_loop = asyncio.get_event_loop()
+                my_loop.create_task(coro(1, 2))
+                """,
+            ),
+        ],
+    )
+    def test_import(self, tmpdir, input_code, expected_output):
+        self.run_and_assert(tmpdir, input_code, expected_output)
+
+    @pytest.mark.parametrize(
+        "input_code,expected_output",
+        [
+            (
+                """
+                from asyncio import Task
+                Task(coro(1, 2))
+                """,
+                """
+                import asyncio
+
+                asyncio.create_task(coro(1, 2))
+                """,
+            ),
+            (
+                """
+                from asyncio import Task, get_event_loop
+                my_loop = get_event_loop()
+                Task(coro(1, 2), loop=my_loop)
+                """,
+                """
+                from asyncio import get_event_loop
+                my_loop = get_event_loop()
+                my_loop.create_task(coro(1, 2))
+                """,
+            ),
+        ],
+    )
+    def test_from_import(self, tmpdir, input_code, expected_output):
+        self.run_and_assert(tmpdir, input_code, expected_output)
+
+    @pytest.mark.parametrize(
+        "input_code,expected_output",
+        [
+            (
+                """
+                from asyncio import Task as taskInit
+                taskInit(coro(1, 2))
+                """,
+                """
+                import asyncio
+
+                asyncio.create_task(coro(1, 2))
+                """,
+            ),
+            (
+                """
+                from asyncio import get_event_loop, Task as taskInit
+                my_loop = get_event_loop()
+                taskInit(coro(1, 2), loop=my_loop)
+                """,
+                """
+                from asyncio import get_event_loop
+                my_loop = get_event_loop()
+                my_loop.create_task(coro(1, 2))
+                """,
+            ),
+        ],
+    )
+    def test_import_alias(self, tmpdir, input_code, expected_output):
+        self.run_and_assert(tmpdir, input_code, expected_output)
+
+    @pytest.mark.parametrize(
+        "input_code,expected_output",
+        [
+            (
+                """
+                import asyncio
+                asyncio.Task(coro(1, 2), name='task', eager_start=True)
+                """,
+                """
+                import asyncio
+                asyncio.create_task(coro(1, 2), name='task', eager_start=True)
+                """,
```

Comment:
This changes the intended behavior of the code. The `eager_start` argument has seemingly no effect on `create_task`.

I think the `ayncio.eager_task_factory` is the intended way to create eager tasks.
https://docs.python.org/3/library/asyncio-task.html#eager-task-factory

Reply:
ah interesting. I only used eager_task as a kwarg test example to demonstrate we correctly handle kwargs. I'll see if I use another kwarg or if we need to special case for eager_task

Reply:
@andrecsilva could you double check my understanding of the docs, since it's a tiny bit confusing. Seems like eager_start is new so not a ton of use cases yet out there.

So if I understand correctly, if you want to create a `Task()` with eager_start=True, the code would be
```
loop = asyncio.get_running_loop()
task = asyncio.Task(my_coroutine(), loop=loop, eager_start=True)
```
(must pass in a loop otherwise it won't work)

Our replacement could either be
```
  loop = asyncio.get_running_loop()
  loop.set_task_factory(asyncio.eager_task_factory)
  task = loop.create_task(my_coroutine())
```
which is kinda what the docs suggest? but this also works:

```
  loop = asyncio.get_running_loop()
  task = asyncio.eager_task_factory(loop, my_coroutine())
```
though I"m not 100% sure they're totally equivalent.

I'm a tiny bit leaning to not support this right now if we aren't sure, but maybe I'm missing something today.

Reply:
That is also my understanding from reading the documentation.